app/javascript/gobierto_data/lib/router.js

```diff
@@ -27,7 +27,7 @@ export const router = new VueRouter({
       }, {
         path: ":id/q/:queryId?",
         component: Dataset,
-        name: 'Dataset',
+        name: 'Query',
```

Comment:
Because we only use this route to render a saved query, also I need one way to see when the user loads an entire Dataset or only load a saved query belongs to Dataset.